sigs.yaml

```diff
@@ -3870,25 +3870,31 @@ committees:
     - github: BenTheElder
       name: Benjamin Elder
       company: Google
+      email: bentheelder@google.com
     - github: aojea
       name: Antonio Ojea
       company: Google
+      email: antonio.ojea.garcia@gmail.com
     - github: justaugustus
       name: Stephen Augustus
       company: Cisco
+      email: k8s@auggie.dev
     - github: pacoxu
       name: Paco Xu 徐俊杰
       company: DaoCloud
+      email: roollingstone@gmail.com
```

Comment:
I plan to switch my email to Gmail because my company email has been repeatedly rejected by cncf.io and Gmail recently.

Reply:
https://github.com/kubernetes/k8s.io/pull/7663